go.mod

```
@@ -1,6 +1,6 @@
 module github.com/calyptia/go-s3-client
 
-go 1.20
+go 1.21.0
```

It seems to be complaining about this
```suggestion
go 1.21
```